File: src/test/java/br/com/newObjects/model/entities/classes/NovoProduto.java
package br.com.newObjects.model.entities.classes;

import br.com.model.entities.classes.Produto;

public class NovoProduto {

    public static Produto novoProduto() {
        return new Produto("Abacate");
    }

    public static Produto novoProduto2() {
        return new Produto("Chocolate");
    }
}

File: src/test/java/br/com/newObjects/model/entities/classes/NovoProduto.kt
package br.com.newObjects.model.entities.classes

import br.com.model.entities.classes.Produto

object NovoProduto {
    @JvmStatic
    fun novoProduto(): Produto {
        return Produto("Abacate")
    }

    fun novoProduto2(): Produto {
        return Produto("Chocolate")
    }
}
